include/javabind/local.hpp

```diff
@@ -325,7 +325,7 @@ namespace javabind
             return StaticField(_env, _ref, name, signature);
         }
 
-        LocalObjectRef getStaticObjectField(const char* name, const std::string_view& signature)
+        LocalObjectRef getStaticObjectField(const std::string_view& name, const std::string_view& signature)
```

Comment:
I think we should overload (like with `getStaticField`) to avoid constructing an `std::string_view` when we have a string literal (`const char*`) that we would pass directly to JNI.

Reply:
Added overload